### PR TITLE
Evaluating TrecQA systems

### PR DESCRIPTION
**Pull request by rosequ**
- made the QueryParser more forgiving by ignoring the Wildcard characters in the query
- switched to `compareTo()` to handle ordering of -0.0 and Nan
- an evaluation script for evaluating TrecQA systems

### CODE REVIEW

File: src/main/python/evaluate.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+import os
+import re
+
+from nltk.tokenize import TreebankWordTokenizer
+from pyserini import Pyserini
+from castorini_smmodel_bridge import SMModelBridge
+
+
+path_to_castorini = "/Users/royalsequeira/Castorini"  # change this path
+model = SMModelBridge(path_to_castorini + '/models/sm_model/sm_model.no_ext_feats',
```
Comment (lintool) [inline src/main/python/evaluate.py]: don't hard code this.

**Comment by lintool** [comment]
Someone else please look at Python code.
Please write a README on how to user this - we need to start up the bridge, for example?

File: src/main/python/evaluate.py (hunk 1 above)
Comment (rosequ) [inline src/main/python/evaluate.py]: oh sorry, forgot to change it

**Comment by rosequ** [comment]
Can someone do a CR for this one, please...

**Comment by rosequ** [comment]
@gauravbaruah @salman1993 @HTAustin please do a CR on this and let's have it shipped.

**Pull request merged**